packages/typespec-client-generator-core/src/public-utils.ts

```diff
@@ -360,7 +360,7 @@ function findContextPath(
         return result;
       }
     }
-    for (const og of listRawSubClients(context, client)) {
+    for (const og of listOperationGroups(context, client, true)) {
```

Comment:
why are we switching to `listOperationGroups`?

Reply:
these two functions have the same functionality. i removed one of them.